README.md

```diff
@@ -109,6 +109,12 @@ or if you're using with babel-loader, you can
 }
 ```
 
+**If you want to use aria attributes in your SVGs**, set this SVGO plugin option:
+
+```js
+{ removeUnknownsAndDefaults: false }
```

Comment:
@boopathi: I had a PR merged into SVGO that will change this default behavior: https://github.com/svg/svgo/pull/741. Maybe we should wait a few days to see if that fix is released, then we can remove this caveat altogether.